dev.slimevr.SlimeVR.metainfo.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+dev.slimevr.SlimeVR.metainfo.xml by SlimeVR contributors
+
+To the extent possible under law, the person who associated CC0 with
+dev.slimevr.SlimeVR.metainfo.xml has waived all copyright and related or neighboring rights
+to dev.slimevr.SlimeVR.metainfo.xml.
+
+You should have received a copy of the CC0 legalcode along with this
+work.  If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+-->
+<component type="desktop-application">
+  <id>dev.slimevr.SlimeVR</id>
+
+  <name>SlimeVR</name>
+  <summary>An app for facilitating full-body tracking in virtual reality</summary>
+
+  <!-- CC0 so attribution is not required -->
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MIT OR Apache-2.0</project_license>
+
+  <content_rating type="oars-1.1" />
+  <url type="homepage">https://slimevr.dev/</url>
+  <url type="bugtracker">https://github.com/SlimeVR/SlimeVR-Server/issues</url>
+  <url type="faq">https://docs.slimevr.dev/server-setup/slimevr-setup.html</url>
```

Comment:
Is this the best link we have for this? I imagine an alternative could be https://docs.slimevr.dev/slimevr101.html.